examples/python/log_file/requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy
```

Comment:
stupid question: is this actually needed? I know rerun uses lots of numpy internally but there's none of that in the sample

Reply:
It's not, I just copy pasted the `minimal` example as a starting point :smiling_face_with_tear:

Reply:
rerun-sdk already depends on numpy so it will be installed any way. I'd say if the example doesn't include numpy it shouldn't be in the requirements.